main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/gmo-personal/coding_challenge/database"
+	"github.com/gmo-personal/coding_challenge/model"
+	"github.com/gmo-personal/coding_challenge/server"
+	"github.com/gmo-personal/coding_challenge/server/utils"
+	"golang.org/x/crypto/bcrypt"
+)
+
+func main() {
+	database.InitDatabase()
+	err := AddBaseAccountIfNotExists()
+	if err != nil {
+			utils.LogError(err)
```

Comment:
the formatting looks odd here
please run `gofmt` or `goimports` on all files in the repo

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

I was following this indent error flow, please advise.

Reply:
I think @awly is pointing out that you didn't run `gofmt` on this file. This line has 3 tabs when it should be 2.

```go
			utils.LogError(err)
```

Reply:
I will run gofmt